README.md

```diff
@@ -69,27 +69,27 @@ settings = MySettings(
     settings = MySettings(prefix='MYPROJECT_')
     ```
     
-- ***dotenv_path***: Путь до env-файла. Дефолтное значение ```None```. Используется для загрузки переменных из env-файла в окружение. Если dotenv_path ```None``` -  то пойдет по дереву каталогов вверх, ища указанный файл - по умолчанию имя файла для поиска - ```.env```.
-- ***override_env***: ```True``` - перезаписать значения, которые при загрузке из ```.env``` встречаются с такими же именами в окружении, ```False``` - не перезаписывать. Дефолтное значение - ```False```. 
-- ***yaml_settings_path***: Путь до конфигурационного yaml файла. Дефолтное значение ```None```.
-- ***use_env***: ```True``` - использовать переменные окружения среды. Дефолтное значение ```True```.
+- ***dotenv_path***: Path to env-file. Default - ```None```. Using for exporting variables from env-file to environment. If ```dotenv_path``` is ```None``` -  walking up the directory tree looking for the specified file - called ```.env``` by default.
+- ***override_env***: ```True``` - override existing system environment variables, ```False``` - do not override. Default - ```False```. 
+- ***yaml_settings_path***: Path to yaml config file. Default - ```None```.
+- ***use_env***: ```True``` - use environment variables. Default - ```True```.
 
-### Исключения
-***ValueError***: Если тип modules не ```list``` или ```NoneType```. Если тип элемента в ***modules*** не ```ModuleType```.
+### Exceptions
+***ValueError***: If ***modules*** type is not ```list``` or ```NoneType``` and if type of element in ***modules*** is not ```ModuleType```.
 
-Загрузка настроек
------------------
-Загрузку настроек можно инициировать в любом месте проекта.
+Settings loading
+----------------
+Loading settings can be initiated anywhere in the project.
 ```python
 from where_your_settings import settings
 
 settings.init()
 ```
-При повторном вызове снова пройдет по конфигурационным файлам и обновит их.
+If called again, it goes through the configuration files and update properties.
```

Comment:
MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines
```suggestion

If called again, it goes through the configuration files and update properties.
```